_data/templates.json

```diff
@@ -1355,6 +1365,39 @@
 	},
 	"a11yGuidance": "No accessibility guidance.",
 	"variations": [
+		{
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
```

Comment:
Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"